.github/workflows/ci.yml

```diff
@@ -10,11 +10,14 @@ jobs:
   test:
     name: Lint and test
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [20, 22, 24]
```

Comment:
if you want to use this approach, please switch to the one we use in e.g. https://github.com/nearform/fast-jwt